### ISSUE TITLE
Error just after launching the generator

### ISSUE THREAD

**Issue by maxleiko** [jeanparpaillon/generator-saloon#1]
``` sh
yo saloon

events.js:141
      throw er; // Unhandled 'error' event
      ^
AssertionError: Trying to copy from a source that does not exist: /home/leiko/.nvm/versions/node/v4.2.4/lib/node_modules/generator-saloon/templates/.yo-rc.json
    at EditionInterface.exports._copySingle (/home/leiko/.nvm/versions/node/v4.2.4/lib/node_modules/generator-saloon/node_modules/mem-fs-editor/actions/copy.js:45:3)
    at EditionInterface.exports.copy (/home/leiko/.nvm/versions/node/v4.2.4/lib/node_modules/generator-saloon/node_modules/mem-fs-editor/actions/copy.js:23:17)
    at generators.Base.extend.copy_or_render (/home/leiko/.nvm/versions/node/v4.2.4/lib/node_modules/generator-saloon/app/index.js:18:12)
    at module.exports.Saloon.extend.writing (/home/leiko/.nvm/versions/node/v4.2.4/lib/node_modules/generator-saloon/app/index.js:58:11)
    at Object.<anonymous> (/home/leiko/.nvm/versions/node/v4.2.4/lib/node_modules/generator-saloon/node_modules/yeoman-generator/lib/base.js:436:25)
    at /home/leiko/.nvm/versions/node/v4.2.4/lib/node_modules/generator-saloon/node_modules/run-async/index.js:24:25
    at /home/leiko/.nvm/versions/node/v4.2.4/lib/node_modules/generator-saloon/node_modules/yeoman-generator/lib/base.js:448:8
    at processImmediate [as _immediateCallback] (timers.js:383:17)
```

After a fresh install.

**Comment by jeanparpaillon** [comment]
The version on github was not in sync with npmjs.com. Should be fixed now.

**Comment by maxleiko** [comment]
Not working.

**Comment by maxleiko** [comment]
@jeanparpaillon

**Comment by jeanparpaillon** [comment]
Mmmh, forgot to include 'templates' dir in package.json.
I was supposing all files were included by default.

**Comment by jeanparpaillon** [comment]
Should be fixed in latest published version: 0.1.3

**Comment by maxleiko** [comment]
Works now. Thanks
